### ISSUE TITLE
[MacOS_ARM] BLUS30425 & BLUS30540 stuck after boot

### ISSUE THREAD

**Issue by Ibxx0403** [RPCS3/rpcs3#16435]
### Quick summary

The following TT games gets stuck/freezes shortly after boot.
LEGO® Indiana Jones™ 2: The Adventure Continues [BLUS30425] 
LEGO® Star Wars® III: The Clone Wars™ [BLUS30540]

### Details

As detailed above, both games worked before switching to MacOS arm build. 
![image](https://github.com/user-attachments/assets/09fb886b-7d8d-44a4-a2f5-c7d769812d80)
![image](https://github.com/user-attachments/assets/bd019277-cb30-42e6-b693-c5de50923e66)

### Build with regression

0.0.34-17210-0fd6956b Alpha

### Attach two log files

[star_not_RPCS3.log](https://github.com/user-attachments/files/18211529/star_not_RPCS3.log)
[in_not_RPCS3.log](https://github.com/user-attachments/files/18211530/in_not_RPCS3.log)
[in_working RPCS3.log](https://github.com/user-attachments/files/18211531/in_working.RPCS3.log)
[star_working RPCS3.log](https://github.com/user-attachments/files/18211532/star_working.RPCS3.log)

### System configuration

Macbook Pro M1Pro

**Comment by mavethee** [comment]
@schm1dtmac yeah it's a bug like VSH or anything firmware related on ARM builds.

@Ibxx0403 Retest them on x86_64 build, if they still work with 17210 and up, that's not regression.

**Comment by kd-11** [comment]
Yea, we need to figure out if this is arm64-specific. Also disable some speedhacks like the async host memory manager in advanced tab that has been reported to cause some hangs/freezes.

**Comment by ibxx0402** [comment]
I have tried turning off async host memory manager, and it still gets stuck on the skull animation. The animation still functions but it never gets past the animation.  I tried running the game on intel build again at newest build and it still functions.

**Comment by Ibxx0403** [comment]
i tested once again on the latest build, this time the it goes past the screens, but It takes a long time and performance is poor. Lego Indiana Jones 2 is about 7-9 fps. While Lego Star Wars similar but a bit lower. 
[BLUS30540.log](https://github.com/user-attachments/files/18301333/BLUS30540.log)
[BLUS30425.log](https://github.com/user-attachments/files/18301334/BLUS30425.log)

**Comment by mavethee** [comment]
Similar issue like with DiRT 🤔

**Comment by digant73** [comment]
probably still an issue, but please @mavethee, @ibxx0402 provide an update

**Comment by mavethee** [comment]
@digant73 still an issue

**Comment by ibxx0402** [comment]
The last couple of times i have tested the issue persisted, however i have since updated to MacOS 26.1 beta, which seems to have broken all the RPCS3 games i have tested. They all crash on boot.

i have attached the error log, from MacOS since it seems the crash happens before rpcs3 generate a log
[rpcs3 error.txt](https://github.com/user-attachments/files/23153770/rpcs3.error.txt)

**Comment by digant73** [comment]
please an update on current build
